main.py

```diff
@@ -364,6 +378,18 @@ def update_game(self):
             self.bullet_last_held_long = False
             self.continous_bullets_message = False
 
+        if pyxel.btnr(pyxel.KEY_Z):
+            if self.used_special_move: return
+            self.used_special_move = True
+            # some cool animations
+            for enem in self.enemies:
+                if enem.is_alive and enem.show:
+                    enem.hit_special_move()
+
+            if self.monster.show and self.monster.alive:
+                self.monster.hit_special_move()
+
+

```

Comment:
Here, Flake8 proposes 2 faults: `multiple statements on one line (colon)`, which can be fixed easily in this case, and `too many blank lines (2)`. I propose this change:

```suggestion
        if pyxel.btnr(pyxel.KEY_Z):
            if self.used_special_move:
                return
            self.used_special_move = True
            # some cool animations
            for enem in self.enemies:
                if enem.is_alive and enem.show:
                    enem.hit_special_move()

            if self.monster.show and self.monster.alive:
                self.monster.hit_special_move()

```